package.json

```
@@ -88,18 +88,18 @@
     "@docusaurus/preset-classic": "^2.4.0",
     "@stylelint/prettier-config": "^2.0.0",
     "@stylelint/remark-preset": "^4.0.0",
-    "eslint": "^8.38.0",
+    "eslint": "^8.39.0",
     "eslint-config-stylelint": "^18.0.0",
     "eslint-plugin-react": "^7.32.2",
-    "glob": "^10.1.0",
+    "glob": "^10.2.2",
     "husky": "^8.0.3",
     "lint-staged": "^13.2.1",
     "npm-run-all": "^4.1.5",
     "postcss": "^8.4.21",
     "prettier": "^2.8.7",
     "remark": "^14.0.2",
     "remark-cli": "^11.0.0",
-    "stylelint": "https://github.com/stylelint/stylelint/tarball/15.5.0",
+    "stylelint": "https://github.com/stylelint/stylelint/tarball/HEAD",
```

Waiting for `15.6.0`:

```suggestion
    "stylelint": "https://github.com/stylelint/stylelint/tarball/15.6.0",
```